access/src/test/resources/server.properties

```diff
@@ -58,10 +58,6 @@ solr.context=solr
 # Access control
 access.group.admin=admin
 
-# Java Message Server (localhost, 61616)
-jms.host=localhost
-jms.port=:61617
```

Comment:
odd that this port number didn't match up with the comment, or with the jms.port in deposit.properties

Reply:
Yeah, one got changed without the other being updated.  I believe we are using 61617 for the VM, which is slightly off from the default jms port.